guides/common/modules/proc_troubleshooting-permission-issues.adoc

```diff
@@ -0,0 +1,26 @@
+[id="troubleshooting-permission-issues"]
+= Troubleshooting permission issues
+
+You can perform a health check on {ProjectServer} to find permission issues.
```

Comment:
"Permission issues" can mean a lot of things, not just fixing duplicate permissions. So using "permission issues" in the heading and intro here is too vague and can be misleading. Can you consider a more precise description of what the section contains?

Reply:
The revised introduction along with the move to Upgrading provide sufficient context. You can resolve this thread.